spark-3.3/clickhouse-spark/src/main/scala/xenon/clickhouse/Constants.scala

```diff
@@ -19,14 +19,15 @@ object Constants {
   //////////////////////////////////////////////////////////
   //////// clickhouse datasource catalog properties ////////
   //////////////////////////////////////////////////////////
-  final val CATALOG_PROP_HOST      = "host"
-  final val CATALOG_PROP_GRPC_PORT = "grpc_port"
-  final val CATALOG_PROP_HTTP_PORT = "http_port"
-  final val CATALOG_PROP_PROTOCOL  = "protocol"
-  final val CATALOG_PROP_USER      = "user"
-  final val CATALOG_PROP_PASSWORD  = "password"
-  final val CATALOG_PROP_DATABASE  = "database"
-  final val CATALOG_PROP_TZ        = "timezone" // server(default), client, UTC+3, Asia/Shanghai, etc.
+  final val CATALOG_PROP_HOST          = "host"
+  final val CATALOG_PROP_GRPC_PORT     = "grpc_port"
+  final val CATALOG_PROP_HTTP_PORT     = "http_port"
```

Comment:
would you mind to also standard `tcp_port` since `NodeSpec` expects is specifically?

Reply:
The https://github.com/ClickHouse/clickhouse-jdbc does not support TPC protocol yet